frontend/packages/ux-editor/src/components/FormComponent.test.tsx

```diff
@@ -35,13 +35,12 @@ describe('FormComponent', () => {
     expect(screen.getByRole('button', { name: textMock('general.delete') })).toBeInTheDocument();
   });
 
-  it('should delete when clicking the Delete button', async () => {
+  it('should confirm delete when clicking the Delete button', async () => {
+    jest.spyOn(window, 'confirm').mockReturnValue(true);
     await render();
-
     const button = screen.getByRole('button', { name: textMock('general.delete') });
     await act(() => user.click(button));
-
-    expect(queriesMock.saveFormLayout).toHaveBeenCalledTimes(1);
```

Comment:
I think we should add a new test that tests the deletion of a component and that checks that `saveFormLayout` has been called

Reply:
done